### PR TITLE
Stylesheet changes

### PR DESCRIPTION
**Pull request by archanaknayanar**

### CODE REVIEW

**Comment by archanaknayanar** [comment]
Missing tag is solved in new commit.

**Comment by midhunkrishna** [comment]
@archanaknayanar please correct the indentation of the html file.

File: sample.css [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+#header {
+	background-color: #66CCFF;
+	color: white;
+	text-align: center;
+	padding: 10px;
+}
+#content {
+	padding-left:  : 10px;
+}
+#nav {
```
Comment (midhunkrishna) [inline sample.css]: your id is Nav in html and nav here. it should be the same

File: sample.css [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+	font-family: Helvetica;
+}
+a {
+	text-decoration: none;
+	
+}
+h1 h2 h3 {
+	margin: 0;
+}
+#container {
```
Comment (midhunkrishna) [inline sample.css]: uppercase lowercase issue.

**Pull request merged**